ui/app/AppLayouts/Chat/popups/PaymentRequestModal.qml

```diff
@@ -83,6 +78,7 @@ StatusDialog {
         readonly property ModelEntry selectedHolding: ModelEntry {
```

Comment:
This is marked as required in:

https://github.com/status-im/status-desktop/blob/6d59284bc29b281be9800cd42323d06bd4add3ba/ui/StatusQ/include/StatusQ/modelentry.h#L18